src/__tests__/Carousel.js

```diff
@@ -5,6 +5,10 @@ import renderer from 'react-test-renderer';
 import * as index from '../index';
 import Swipe from 'react-easy-swipe';
 
+window.requestAnimationFrame = fn => {
```

Comment:
Please, add some unit tests...

Reply:
I can see the need for a few tests here:
- Check if the items were added in the right place;
- Check if the carousel behaves correctly with 0, 1, 2 or 3 items;
- Check if you can change the number of children of the carousel;
- Any intermediate state required for swiping and clicking next/prev;

These are just some ideas. Feel free to implement what you think would make sense.